linkerd/error.go

```diff
@@ -130,3 +147,13 @@ func ErrProcessOAM(err error) error {
 func ErrApplyHelmChart(err error) error {
 	return errors.New(ErrApplyHelmChartCode, errors.Alert, []string{"error applying helm chart"}, []string{err.Error()}, []string{}, []string{})
 }
+
+// ErrAddonFromHelm is the error for installing addons through helm chart
+func ErrAddonFromHelm(err error) error {
+	return errors.New(ErrAddonFromHelmCode, errors.Alert, []string{"Error with addon install operation by helm chart"}, []string{err.Error()}, []string{"The helm chart URL in additional properties of addon Operation might be incorrect", "The helm installation failed due to any other reason"}, []string{})
```

Comment:
```suggestion
	return errors.New(ErrAddonFromHelmCode, errors.Alert, []string{"Error with addon install operation by helm chart"}, []string{err.Error()}, []string{"The helm chart URL in additional properties of addon operation might be incorrect", "The helm installation failed due to any other reason"}, []string{})
```

Reply:
What does this mean? "The helm installation failed due to any other reason" - can we rephrase or provide an alternate suggestion?